fhir-server-test/src/test/java/com/ibm/fhir/server/test/SortingTest.java

```diff
@@ -279,7 +279,7 @@ public void testCreateObservation5() throws Exception {
     // Patient?gender=male&_sort:asc=family
     @Test(groups = { "server-search" }, dependsOnMethods = { "testCreatePatient1",
             "testCreatePatient2", "testCreatePatient3", "testCreatePatient4",
-            "testCreatePatient5" })
+    "testCreatePatient5" })
```

Comment:
why this bad format change?

Reply:
this is by my eclipse auto actions during saving, I would think this is actually the correct format, because ")" at the end is the most top level.

Reply:
why would line 281 be indented and not 282?

Reply:
and this is why i don't like auto-formatting...

Reply:
281 is for "}", but any way, I can make it better

Reply:
done